.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+script:
+  - mkdir build && cd build && cmake .. && cmake --build .
+matrix:
+  include:
+    compiler:
+      - gcc
+      # clang
+    addons:
+      apt:
+        sources:
+          - ubuntu-toolchain-r-test
+        packages:
+          - libboost-filesystem-dev
+          - libboost-program-options-dev
+          - txt2tags
+          #- g++-4.8
+          - g++-7
+          #- libstdc++6-4.7-dev
+    env:
+      - MATRIX_EVAL="CC=gcc-7 && CXX=g++-7"
+before_install:
+  - eval "${MATRIX_EVAL}"
+install:
+  - g++ -v
```

Comment:
What's the point of this line?
Looking in the output of the build from your fork I see it reports the local g++ (4.8) instead of v7 you chose to use here.
(https://travis-ci.org/finikorg/thunderbolt-software-user-space/builds/316323032?utm_source=github_status&utm_medium=notification line 660)

Reply:
That was g++ check since default compiler is a bit old for the project. I wanted to know why build was failing.

Reply:
Oh, so no need to get it in to the project, isn't it?
Do you want to remove it or I can do it on the merge?

Reply:
Fixed